versions.tf

```diff
@@ -4,8 +4,7 @@ terraform {
       source  = "clumio-code/clumio"
       version = "~>0.3.0"
```

Comment:
Change the version to ">=0.3.0" as we will soon be releasing version 0.4.0. If we have it as "~>0.3.0" it will only with 0.3.x.